packages/hoprd/src/index.ts

```diff
@@ -380,6 +380,10 @@ async function main() {
     }
   }
 
+  const messageAcknowledged = (ackChallenge: string): void => {
```

Comment:
All message acknowledgments for now globally logged, so they can make it over the WS.